libs/fixtures/data/electionMinimalExhaustiveSample/cvrFiles/cdf_cast_vote_record_report.json

```diff
@@ -0,0 +1,400 @@
+                  ]
+                }
+              ]
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "@type": "CVR.CVR",
+      "BallotStyleId": "2F",
+      "BallotStyleUnitId": "precinct-1",
+      "PartyIds": ["1"],
+      "CreatingDeviceId": "scanner-1",
+      "ElectionId": "some-id",
+      "BatchId": "batch-2",
+      "UniqueId": "id-169",
+      "vxBallotType": "absentee",
+      "vxTestBallot": true,
+      "CurrentSnapshotId": "id-169-original",
+      "CVRSnapshot": [
+        {
+          "@id": "id-169-original",
+          "@type": "CVR.CVRSnapshot",
+          "Type": "original",
+          "CVRContest": [
+            {
+              "@type": "CVR.CVRContest",
+              "ContestId": "best-animal-fish",
+              "CVRContestSelection": [
+                {
+                  "@type": "CVR.CVRContestSelection",
+                  "ContestSelectionId": "seahorse",
+                  "SelectionPosition": [
+                    {
+                      "HasIndication": "yes",
+                      "NumberVotes": 1
+                    }
+                  ]
+                },
+                {
+                  "@type": "CVR.CVRContestSelection",
+                  "ContestSelectionId": "salmon",
+                  "SelectionPosition": [
+                    {
+                      "HasIndication": "yes",
+                      "NumberVotes": 1
+                    }
+                  ]
+                }
+              ],
+              "Overvotes": 1
+            },
+            {
+              "@type": "CVR.CVRContest",
+              "ContestId": "aquarium-council-fish",
+              "CVRContestSelection": [
+                {
+                  "@type": "CVR.CVRContestSelection",
+                  "ContestSelectionId": "manta-ray",
+                  "SelectionPosition": [
+                    {
+                      "HasIndication": "yes",
+                      "NumberVotes": 1
+                    }
+                  ]
+                }
+              ],
+              "Undervotes": 1
+            },
+            {
+              "@type": "CVR.CVRContest",
+              "ContestId": "fishing",
+              "CVRContestSelection": [
+                {
+                  "@type": "CVR.CVRContestSelection",
+                  "ContestSelectionId": "no",
+                  "SelectionPosition": [
+                    {
+                      "HasIndication": "yes",
+                      "NumberVotes": 1
+                    }
+                  ]
+                }
+              ]
+            }
+          ]
+        }
+      ]
+    },
+    {
+      "@type": "CVR.CVR",
+      "BallotStyleId": "2F",
+      "BallotStyleUnitId": "precinct-1",
+      "PartyIds": ["1"],
+      "CreatingDeviceId": "scanner-1",
+      "ElectionId": "some-id",
+      "BatchId": "batch-2",
+      "UniqueId": "id-172",
+      "vxBallotType": "absentee",
+      "vxTestBallot": true,
+      "BallotImage": [
```

Comment:
This one has a `BallotImage` because there is a write-in. You can also include a `WriteInImage` next to the particular write in but this seems like a more appropriate place to include it given we include the whole image.

Reply:
agree with the choice here